react/I18n/format.jsx

```diff
@@ -2,9 +2,14 @@ import format from 'date-fns/format'
 import { DEFAULT_LANG } from '.'
 
 export const initFormat = (lang, defaultLang = DEFAULT_LANG) => {
-  const locales = {
-    [defaultLang]: require(`date-fns/locale/${defaultLang}/index.js`)
+  const locales = {}
+
+  try {
+    locales[defaultLang] = require(`date-fns/locale/${defaultLang}/index.js`)
+  } catch (err) {
+    console.warn(`The "${defaultLang}" locale isn't support by date-fns`)
```

Comment:
```suggestion
    console.warn(`The "${defaultLang}" locale isn't supported by date-fns or has not been included in the build. Check if you have configured a ContextReplacementPlugin that is too restrictive.`)
```

I think our error messages too often do not provide a way for the user (developer in this case) to understand clearly what is going on or provide a suggestion.